### PR TITLE
Adding sugestions to install WSL2 and it packages in environment-setup.md

### PR DESCRIPTION
**Pull request by BeAmazedVariable**
## Type of Change

- [ ] **Bugfix**: Change which fixes an issue
- [ ] **New Feature**: Change which adds functionality
- [x] **Documentation Update**: Change which improves documentation
- [ ] **UI**: Change which improves UI

## Description

When I try to install redis using my newly installed WSL2, the installation was not succedded since my WSL2 have not updated yet. Running `sudo apt update` will get your WSL2 update so I think adding it in the doc might help other newly contributors as well.
## Checklist

- [ ] **Quality**: This PR builds and passes our npm test and works locally
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [x] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

**Review by Grommers00 — DISMISSED** [review]
Looks good!

File: docs/environment-setup.md [hunk 1]
```diff
@@ -37,6 +37,31 @@
...
+
+```bash
+# fetch updated packages information
+sudo apt update
+
+# upgrade or install outdated packages
+sudo apt upgrade
+```
+
+2. Depending on the distribution you chose for WSL2, install redis using that distro's package manager. You can check your distro's package manager [here](https://www.tecmint.com/linux-distro-for-power-users/). The two most common package manager are APT and DNF.
```
[the remark this answers is not in the document]
Comment (BeAmazedVariable) [reply docs/environment-setup.md]: Got it !

**Review by humphd — DISMISSED** [review]

**Review by Grommers00 — APPROVED** [review]

**Comment by Grommers00** [comment]
Just gotta rebase it and we can merge :D

**Comment by BeAmazedVariable** [comment]
@Grommers00 I just rebase my commit with the lastest commit on the master

**Review by Grommers00 — APPROVED** [review]

**Review by humphd — APPROVED** [review]

**Pull request merged**